pylintrc

```diff
@@ -0,0 +1,426 @@
+[MASTER]
```

Comment:
Can you remove this file? We are using flake8 not pylint.

Reply:
ok you did it.
At first I didn't know about pylint vs flake so I wanted to replicate onmt-tf.
This explains also the dummy comments below in each method/function/class